apps/admin-portal/src/components/applications/wizard/saml-protcol-settings-all-option-wizard-form.tsx

```diff
@@ -202,44 +200,76 @@ export const SAMLProtocolAllSettingsWizardForm: FunctionComponent<SAMLProtocolAl
                             </Hint>
                         </Grid.Column>
                     </Grid.Row>
-                    { (configureMode === SAMLConfigModes.MANUAL) &&
-                    <Grid.Row columns={ 1 }>
-                        <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 8 }>
-                            <Field
-                                name="issuer"
-                                label="Issuer"
-                                required={ true }
-                                requiredErrorMessage="Please provide the issuer"
-                                type="text"
-                                placeholder={ "Enter the issuer name" }
-                                value={ initialValues?.inboundProtocolConfiguration?.saml?.manualConfiguration?.issuer }
-                                readOnly={ initialValues?.saml?.issuer }
-                            />
-                            <Hint>
-                                { `This specifies the issuer. This is the "saml:Issuer" element that contains
-                            the unique identifier of the Application. This is also the issuer value
-                            specified in the SAML Authentication Request issued by the Application. ` }
-                            </Hint>
-                        </Grid.Column>
-                    </Grid.Row>
-                    }
-                    { (configureMode === SAMLConfigModes.MANUAL) &&
-                    < URLInputComponent
-                        urlState={ assertionConsumerUrls }
-                        setURLState={ setAssertionConsumerUrls }
-                        labelName={ "Assertion consumer URLs" }
-                        placeholder={ "Enter url " }
-                        validationErrorMsg={ "Please add valid URL" }
-                        validation={ (value: string): boolean => {
-                            return FormValidation.url(value);
-                        } }
-                        required={ true }
-                        showError={ showAssertionConsumerUrlError }
-                        setShowError={ setAssertionConsumerUrlError }
-                        hint={ "This specifies the assertion Consumer URLs that the browser " +
-                        "should be redirected to after the authentication is successful. " +
-                        "This is the Assertion Consumer Service (ACS) URL of the Application" }
-                    />
+                    {
+                        (configureMode === SAMLConfigModes.MANUAL) &&
```

Comment:
As a good practice always have the constant in this case `SAMLConfigModes.MANUAL` on the left side of the expression.